code/modules/mob/living/carbon/carbon.dm

```diff
@@ -297,6 +297,7 @@
 	if (istype(item, /obj/item/weapon/grab))
 		var/obj/item/weapon/grab/G = item
 		item = G.throw_held() //throw the person instead of the grab
+		qdel(G)
```

Comment:
It doesn't seem that grab's Destroy properly unequips the grab item.
I think it'd be better to call unequipping proc for grab here, it'll qdel in there anyway.

Reply:
It worked when I tested it but I'll change it to unEquip()